File: sdrmobil/lib/models/Raporlar/gunluk_tahsilat.dart
import 'package:intl/intl.dart';

class GunlukTahsilat {
  String fisNo;
  int cariSayac;
  String fisTuru;
  DateTime tarih;
  String aktarimTemsilci;
  double tlAlacak;
  double tlBorc;
  String kasaKodu;
  String sorumlu;
  String cariKodu;
  String cariUnvan;

  GunlukTahsilat(
      {this.fisNo,
      this.cariSayac,
      this.fisTuru,
      this.tarih,
      this.aktarimTemsilci,
      this.tlAlacak,
      this.tlBorc,
      this.kasaKodu,
      this.sorumlu,
      this.cariKodu,
      this.cariUnvan});

  GunlukTahsilat.fromJson(Map<String, dynamic> json) {
    fisNo = json['fis_no'];
    cariSayac = json['cari_sayac'];
    fisTuru = json['fis_turu'];
    tarih = json['tarih'] == null ? null : DateTime.parse(json['tarih'].toString());
    aktarimTemsilci = json['aktarim_temsilci'];
    tlAlacak = json['tl_alacak'];
    tlBorc = json['tl_borc'];
    kasaKodu = json['kasa_kodu'];
    sorumlu = json['sorumlu'];
    cariKodu = json['cari_kodu'];
    cariUnvan = json['cari_unvan'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['fis_no'] = this.fisNo;
    data['cari_sayac'] = this.cariSayac;
    data['fis_turu'] = this.fisTuru;
    data['tarih'] = DateFormat('yyyy-MM-dd 00:00:00.000').format(this.tarih);
    data['aktarim_temsilci'] = this.aktarimTemsilci;
    data['tl_alacak'] = this.tlAlacak;
    data['tl_borc'] = this.tlBorc;
    data['kasa_kodu'] = this.kasaKodu;
    data['sorumlu'] = this.sorumlu;
    data['cari_kodu'] = this.cariKodu;
    data['cari_unvan'] = this.cariUnvan;
    return data;
  }
}

File: SDRMobilApi/SDREntities/GunlukTahsilat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SDRMobilApi.SDREntities
{
    public class GunlukTahsilat
    {
        public string fis_no { get; set; }
        public int cari_sayac { get; set; }
        public string fis_turu { get; set; }
        public DateTime tarih { get; set; }
        public string aktarim_temsilci { get; set; }
        public decimal tl_alacak { get; set; }
        public decimal tl_borc { get; set; }
        public string kasa_kodu { get; set; }
        public string sorumlu { get; set; }
        public string cari_kodu { get; set; }
        public string cari_unvan { get; set; }
    }
}
